test_scripts/oop_poc_file.py

```diff
@@ -50,21 +50,19 @@ def create_random_file(
 async def store_file():
     # Creating annotations for my file ==================================================
     annotations_for_my_file = {
-        "my_key_string": AnnotationsValue(
-            type=AnnotationsValueType.STRING, value=["b", "a", "c"]
-        ),
-        "my_key_bool": AnnotationsValue(
-            type=AnnotationsValueType.BOOLEAN, value=[False, False, False]
-        ),
-        "my_key_double": AnnotationsValue(
-            type=AnnotationsValueType.DOUBLE, value=[1.2, 3.4, 5.6]
-        ),
-        "my_key_long": AnnotationsValue(
-            type=AnnotationsValueType.LONG, value=[1, 2, 3]
-        ),
-        "my_key_timestamp": AnnotationsValue(
-            type=AnnotationsValueType.TIMESTAMP_MS, value=[1701362964066, 1577862000000]
-        ),
+        "my_single_key_string": "a",
+        "my_key_string": ["b", "a", "c"],
+        "my_key_bool": [False, False, False],
```

Comment:
I forget, but what does that convert function do when there are mixed types?

Reply:
It's returns an `object` which is than turned into a string:
`nested_annos[key] = {"type": "STRING", "value": [str(e) for e in elements]}`